.travis.yml

```diff
@@ -111,6 +111,21 @@ before_script:
   - echo -e "[pdns-sqlite3-1]\nDriver = SQLite3\nDatabase = ${PWD}/regression-tests/pdns.sqlite3\n\n[pdns-sqlite3-2]\nDriver = SQLite3\nDatabase = ${PWD}/regression-tests/pdns.sqlite32\n" > ${HOME}/.odbc.ini
   - export GODBC_SQLITE3_DSN=pdns-sqlite3-1
 
+  # ldap-backend test setup
+  - sudo apt-get -qq --no-install-recommends install slapd ldap-utils
+  - mkdir /tmp/ldap-dns
+  - pushd /tmp/ldap-dns
+  - for schema in /etc/ldap/schema/{core,cosine}.schema ${TRAVIS_BUILD_DIR}/modules/ldapbackend/{dnsdomain2,pdns-domaininfo}.schema ; do echo include $schema ; done > ldap.conf
+  - mkdir slapd.d
+  - slaptest -f ldap.conf -F slapd.d
+  - sudo cp slapd.d/cn=config/cn=schema/cn={*dns*.ldif /etc/ldap/slapd.d/cn=config/cn=schema/
+  - sudo chown -R openldap:openldap /etc/ldap/slapd.d/
+  - sudo service slapd restart
+  - popd
+  - sudo mkdir /var/lib/ldap/powerdns
+  - sudo chown openldap:openldap /var/lib/ldap/powerdns
```

Comment:
if `/var/lib/ldap` is already owned by `openldap`, these 2 lines could be rolled into `sudo -u openldap mkdir /var/lib/ldap/powerdns`